package.json

```diff
@@ -15,7 +15,7 @@
     "test": "ava",
     "watch": "npm test --watch",
     "start": "node ./markdownlint.js",
-    "precommit": "xo && npm test"
+    "precommit": "xo --prettier && npm test"
```

Comment:
Shouldn't the boolean be set down below on line 57 config object instead of the cli?

Reply:
Oops, didn't even see that config.